config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get 'message/index', to: 'message#index'
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [Optional] API versioning is important: This provide a way to evolve the API over time without breaking existing clients app.